ra-dec-positions.json

```diff
@@ -0,0 +1,12 @@
+    { "type": "record",
+     "namespace": "gcn.notices",
+     "name": "ra-dec-positions",
+     "fields": [
+                {"name": "grb-ra-j2000",    "type": "double", "doc": "ra of GRB/Transient in J2000 epoch [deg]"},
+                {"name": "grb-ra-current",  "type": "double", "doc": "ra of GRB/Transient in Current epoch [deg]"},
+                {"name": "grb-ra-1950",     "type": "double", "doc": "ra of GRB/Transient in 1950 epoch [deg]"},
```

Comment:
I really do not think that we need to provide precomputed coordinates in epochs other than J2000 (including true of date).